common/lib/xmodule/xmodule/course_metadata_utils.py

```diff
@@ -105,6 +106,22 @@ def has_course_ended(end_date):
     return datetime.now(UTC()) > end_date if end_date is not None else False
 
 
+def course_starts_within(start_date, look_ahead_days):
+    """
+    Given a course's start datetime and look ahead days, returns whether
```

Comment:
This method's doc string is not correct. It should be something like "Given a course's start datetime and look ahead days, returns True if course's start date falls within look ahead days otherwise False"